Makefile

```
@@ -11,10 +13,69 @@ DISABLE_PRIVATE_DNS = ${PWD}/apps/disable-private-dns/src
 UPDATE_DB = ${PWD}/apps/update-db/src
 UPLOAD_LOG = ${PWD}/apps/upload-log/src
 DYNAMO = ${PWD}/lib/dynamo
+ENV_FILE = ${PWD}/hyp3.env
+
+DEFAULT_PROFILE := default
+EXTRACTED_PROFILE := $(shell grep -E '^AWS_DEFAULT_PROFILE=' $(ENV_FILE) | cut -d '=' -f 2 | sed 's/^$$/${DEFAULT_PROFILE}/')
+AWS_DEFAULT_PROFILE ?= $(shell [ -z "$(EXTRACTED_PROFILE)" ] && echo "$(DEFAULT_PROFILE)" || echo "$(EXTRACTED_PROFILE)")
+
+DEFAULT_REGION := us-west-2
+EXTRACTED_REGION := $(shell grep -E '^AWS_DEFAULT_REGION=' $(ENV_FILE) | cut -d '=' -f 2 | sed 's/^$$/$(DEFAULT_REGION)/')
+AWS_DEFAULT_REGION ?= $(shell [ -z "$(EXTRACTED_REGION)" ] && echo "$(DEFAULT_REGION)" || echo "$(EXTRACTED_REGION)")
+
+DEPLOY_ENV_IMAGE_NAME ?= hyp3_deploy
+
+SET_AWS_ACCOUNT_ENV_VARS = export AWS_DEFAULT_PROFILE=${AWS_DEFAULT_PROFILE}; \
+	export AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION}; \
+	export AWS_DEFAULT_ACCOUNT=`aws sts get-caller-identity --query 'Account' --output=text --profile ${AWS_DEFAULT_PROFILE};`
+
+DOCKER_RUN = docker run --rm -it \
+	--env-file $(ENV_FILE) \
+	-v ~/.aws/:/root/.aws/:ro \
+	-v /tmp/aws-cli-cache:/root/.aws/cli/cache \
+	-v ${PWD}:/hyp3/ \
+	-e AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY \
+	-e AWS_DEFAULT_PROFILE \
+	-e AWS_DEFAULT_REGION \
+	-e AWS_DEFAULT_ACCOUNT \
+	${DEPLOY_ENV_IMAGE_NAME}:latest
```

```suggestion
	${DEPLOY_ENV_IMAGE_NAME}:latest
```
I'm not sure we'll want to use the latest tag here, or a specific version number...